server/interpreter.js

```diff
@@ -693,7 +693,7 @@ Interpreter.prototype.initObject_ = function() {
           typeof value === 'string') {
         // No boxed primitives in Code City.
         throw new intrp.Error(state.scope.perms, intrp.TYPE_ERROR,
-            'Boxed primitives not supported.');
```

Comment:
In my experience, JS errors don't have trailing punctuation.

Reply:
And indeed I removed it.